modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js

```diff
@@ -132,23 +133,28 @@ const SidebarPanelInfoView = ({
 								</p>
 
 								{sortedViewURLS.map((language) => (
-									<div
-										className="autofit-row autofit-row-center mb-1"
+									<ClayLayout.ContentRow
+										className="autofit-row-center mb-1"
```

Comment:
You can remove `autofit-row-center` and use api attribute `verticalAlign="center"` 😃

Reply:
I looked the docs and I didn't find it, that's my level of asleepness today

Reply:
ContentRow section:

<img width="863" alt="Screenshot 2020-07-24 at 12 45 22" src="https://user-images.githubusercontent.com/15845466/88383833-8e605100-cdab-11ea-9140-77bfc0081ed2.png">